_events_past/2014-06-09-1st-jlesc-workshop.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page_event
+title: 1st JLESC Workshop
+date: 2016-02-19
+updated:
+navbar: Events
+event_location: Sophia Antipolis, France
+event_start: 2014-06-09
+event_end: 2014-06-11
+event_url:
+event_short_url:
+wayback_url:
```

Comment:
Find the original website using the Web Archive. They have permanent copies of virtually all websites ever created (or close to this), even if they are not online anymore. Check out the site for the 4th workshop!